runtime/onert/core/src/ir/Shape.cc

```diff
@@ -73,10 +73,12 @@ void Shape::extendRank(int to_rank)
 
 uint64_t Shape::num_elements() const
 {
+  constexpr int32_t unspecified_dim = -1;
```

Comment:
@hyunsik-yoon It would be better to define once for `unspecified_dim`. What about defining in `Shape.h`?